packages/fluentui/e2e/tests/dialogInDialog.spec.ts

```diff
@@ -69,20 +75,23 @@ describe('Dialog in Dialog', () => {
 
   it('A click on content and pressing ESC button should close the last opened dialog', () => {
     cy.clickOn(outerTrigger); // opens dialog
+    cy.wait(50);
```

Comment:
can we wrap the `clickOn` + `wait` in a helper function for each of the test suites ?

Reply:
We cannot do it because, there is not only dialog button.